AppCenterDistribute/AppCenterDistributeTests/MSDistributeTests.m

```diff
@@ -876,7 +876,7 @@ - (void)testCheckLatestReleaseOnRecoverableError {
   [self.settingsMock setObject:@1 forKey:kMSUpdateTokenRequestIdKey];
   [self.settingsMock setObject:@1 forKey:kMSPostponedTimestampKey];
   [self.settingsMock setObject:@1 forKey:kMSDistributionGroupIdKey];
-  [self.sut checkLatestRelease:@"token" distributionGroupId:@"groupId" releaseHash:@"releaseHash"];
+  [self.sut checkLatestRelease:@"token" distributionGroupId:@"groupId" releaseHash:kMSTestReleaseHash];
```

Comment:
why stop at introducing the constant for the release hash string, let's add a constant for `groupdID`, too?

Reply:
Added. Thank you.